tests/unittests/CoreGraphics/CGPathTests.mm

```diff
@@ -582,3 +582,72 @@ CGPathRef newPathForRoundRect(CGRect rect, CGFloat radius) {
 
     EXPECT_TRUE(test);
 }
+
+void EXPECT_POINTEQ(CGPoint pathPoint, CGFloat x, CGFloat y) {
+    EXPECT_EQ(x, pathPoint.x);
+    EXPECT_EQ(y, pathPoint.y);
+}
+
+void EXPECT_SIZEEQ(CGSize pathSize, CGFloat width, CGFloat height) {
+    EXPECT_EQ(width, pathSize.width);
+    EXPECT_EQ(height, pathSize.height);
+}
+
+// Simple tests for the status of a CGPath during the CGPath rework into D2D.
+TEST(CGPath, CGPathD2DSimpleCreation) {
+    CGMutablePathRef path = CGPathCreateMutable();
+    EXPECT_NE(nullptr, path);
+
+    CGPathMoveToPoint(path, NULL, 50, 50);
```

Comment:
Lets add some tests that test that.  The previous iteration of code wasnt initializing the points.

---
In reply to: [83301024](https://github.com/Microsoft/WinObjC/pull/1138#discussion_r83301024) [](ancestors = 83301024,83132608)